modules/csg/doc_classes/CSGShape3D.xml

```diff
@@ -89,8 +89,8 @@
 		<member name="operation" type="int" setter="set_operation" getter="get_operation" enum="CSGShape3D.Operation" default="0">
 			The operation that is performed on this shape. This is ignored for the first CSG child node as the operation is between this node and the previous child of this nodes parent.
 		</member>
-		<member name="snap" type="float" setter="set_snap" getter="get_snap" default="0.001">
-			Snap makes the mesh vertices snap to a given distance so that the faces of two meshes can be perfectly aligned. A lower value results in greater precision but may be harder to adjust. The top-level CSG shape's snap value is used for the entire CSG tree.
+		<member name="snap" type="float" setter="set_snap" getter="get_snap" default="0.001" deprecated="Snap is not used by CSG anymore.">
+			CSG no longer uses snap and has been removed because snapping is not implemented in the backing CSG manifold library.
```

Comment:
You could also omit the description technically (it won't count for completion), but we have never done that so far.
```suggestion
		<member name="snap" type="float" setter="set_snap" getter="get_snap" default="0.001" deprecated="The CSG library no longer uses snapping.">
			This property does nothing.
```